sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -128,3 +133,16 @@ export class AESTFileUploadToStudentAPIInDTO {
   @ArrayMinSize(1)
   associatedFiles: string[];
 }
+
+/**
+ * DTO object application summary info.
+ */
+export class ApplicationSummaryAPIOutDTO {
+  applicationNumber: string;
+  studyStartPeriod: string;
+  studyEndPeriod: string;
+  id: number;
+  applicationName: string;
+  submitted?: Date;
+  status: string;
```

Comment:
It is the application status? We can use enum here if so.

Reply:
Changed